app.js

```diff
@@ -104,6 +120,16 @@ function displayFriends() {
         const friendEl = renderFriend(friend);
 
         friendEl.addEventListener('click', () => {
+            if (mushrooms.length === 0) {
+                message = `No mushrooms! Go hunt some more!`;
```

Comment:
you want to execute the code when when the ```if``` check evaluates to true. you want it to be true when there aren't any mushrooms. ```if (!mushrooms.length)``` would also evaluate to true when there aren't any mushrooms: if there aren't any mushrooms, the amount of things in the ```mushrooms``` array will be empty, and if length will be zero. ```0``` is a falsy value in JavaScript so it's not equal to false (```0 !== false```) but it evaluates to false in an ```if``` check